src/com/facebook/buck/android/AndroidBuckConfig.java

```diff
@@ -65,7 +68,7 @@ public Integer getAdbTimeout() {
     return delegate.getValue("ndk", "ndk_repository_path");
   }
 
-  public Optional<String> getNdkAppPlatform() {
+  public Optional<String> getNdkCpuAbiAgnosticAppPlatform() {
```

Comment:
I'd call it `getDefaultAppPlatform`

Reply:
Ok. I thought it might be confusing as it isn't actually setting a default, that happens further down in the stack. Happy to change it though.